tsgen/__main__.py

```diff
@@ -35,29 +35,34 @@ class TelemetrySystemGenerator:
     SENSORS_FILE_NAME = 'sensors.json'
     SCHEMA_FILE_NAME = 'schema.json'
     C_DATABASE_NAME = 'can_database'
+    CAN_S_APP = '_can_s'
+    CAN_C_APP = '_can_c'
```

Comment:
Why `_APP`? Couldn't quite figure out what it means.

Reply:
"Append", Couldn't find a better name, any ideas?